libraries/eosiolib/core/eosio/time.hpp

```diff
@@ -97,6 +100,25 @@ namespace eosio {
         static time_point_sec maximum() { return time_point_sec(0xffffffff); }
         static time_point_sec min() { return time_point_sec(0); }
 
+        static time_point_sec from_iso_string(std::string dateStr){
+           int y,mon,d,h,m;
+           float s;
+           sscanf(dateStr.c_str(), "%d-%d-%dT%d:%d:%fZ", &y, &mon, &d, &h, &m, &s);
```

Comment:
This is not strict enough.  Try strptime.  Also, you need to check the result, as parsing may fail.

Reply:
I added a check for this.